sdks/java/io/google-cloud-platform/pom.xml

```diff
@@ -70,6 +70,30 @@
           </execution>
         </executions>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <goals><goal>analyze-only</goal></goals>
+            <configuration>
+              <!-- Ignore runtime-only dependencies in analysis -->
+              <ignoreNonCompile>true</ignoreNonCompile>
+              <failOnWarning>true</failOnWarning>
+              <usedDependencies>
+                <!--
+                  BEAM-1632: Conflicting classes are brought in by bigtable-protos resulting in
+                  maven-dependency:analyze `unused dependency` error. Marking this as a
+                  `usedDependency` until the issue is resolved.
+                 -->
+                <usedDependency>com.google.api.grpc:grpc-google-common-protos</usedDependency>
```

Comment:
I could not find an easy way to exclude certain classes from a particular (bigtable-protos in this case). Seems like this is a wrong thing to do on part of `bigtable-protos` to include generated classes of protos that it doesn't own and instead should depend on `grpc-google-common-protos`. I would like to discuss with Dan to see if he has any context here.